tox.ini

```diff
@@ -9,6 +9,7 @@ deps =
   pt3: pytest>=3,<4
   pt4: pytest>=4,<5
   pt5: pytest>=5,<6
+  pt6: pytest>=6,<7
```

Comment:
It's probably safe to stop testing on pytest 3 when adding a new one?

Will also help keep the CI matrix manageable.

![image](https://user-images.githubusercontent.com/1324225/146788460-91624896-094f-4039-89d1-6c9512942af5.png)

https://pepy.tech/project/pytest?versions=3.*&versions=4.*&versions=5.*&versions=6.*